hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALProvider.java

```diff
@@ -74,6 +74,22 @@
 
   interface WriterBase extends Closeable {
     long getLength();
+    /**
+     * <pre>
```

Comment:
Do not need to use pre here? These are just normal text.

Reply:
ok, I would fix it.